### ISSUE TITLE
'N Results' bar blocks last line of long notes

### ISSUE THREAD

**Issue by SaeedBaig** [Automattic/simplenote-electron#2544]
When I search for something, the last line of a long note (by "long" I mean has a scrollbar) is blocked or partially blocked by the bar telling you how many results are in that particular note (shown in screenshots below).

### Expected

I expect that the note's contents wouldn't be blocked by other UI elements.

### Observed

The last line of the note is blocked by the 'N Results' bar.

### Reproduced

1. Create a longish note (long enough to have a scrollbar)
2. Search for a word from this note in the Search Bar
3. You should now see the last line of the aforementioned note blocked by the 'N Results' bar.

![bug](https://user-images.githubusercontent.com/29703429/103185451-9a2ee100-4910-11eb-9e71-2f3dab647b54.png)

The only fix is to zoom out til there's no scrollbar, which can be a ridiculous amount for long notes:
![hack](https://user-images.githubusercontent.com/29703429/103185757-e4fd2880-4911-11eb-9f0b-03919ea1ba9e.png)

### Where did you see the bug

- System Make: Dell
- System Model: Dell XPS 13 9360
- OS: Kubuntu (Linux)
- OS version: Kubuntu 20.04.1 LTS
- Simplenote app version: 2.3.0 (desktop app downloaded as deb package from latest release)

**Comment by codebykat** [comment]
Thanks @SaeedBaig, this is captured in #2331 but it's nice to have it as its own issue.

**Comment by pachlava** [comment]
Verified on Windows with Electron 2.5.0-beta1
